src/sql-server-util/src/inspect.rs

```diff
@@ -46,6 +70,37 @@ pub async fn get_max_lsn(client: &mut Client) -> Result<Lsn, SqlServerError> {
     parse_lsn(&result[..1])
 }
 
+/// Returns the maximum log sequence number for the entire database, retrying
```

Comment:
Can we fix the doc here to mention the shortcoming of this method? Namely that the returned value will be less than the maximum LSN of the server since the value depends on the async CDC process. Maybe also link to this thread https://groups.google.com/g/debezium/c/47Yg2r166KM/m/lHqtRF2xAQAJ